packages/test-utils/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Why do we need this annotation?

Reply:
Otherwise `cosmwasm::testing` is leaked into the contract compilation, see this failure https://github.com/babylonlabs-io/cosmos-bsn-contracts/actions/runs/16940450022/job/48007836117.

Ideally, the test-utils is not supposed to be compiled in the WASM compilation, but :shrug: